deploy/cloudformation/iiif-service.yml

```diff
@@ -71,17 +69,16 @@ Parameters:
     Type: Number
     Default: 1
     Description: How many copies of the service task to run
-
-  ServiceURL:
+  SubDomain:
+    Type: String
+    Description: The SubDomain of the service. Combined with DomainName to create the FQDN.
+  CreateDNSRecord:
     Type: String
-    Default: ''
-    Description: The URL of the service
+    Default: "True"
+    Description: Indicate whether the base domain is registered in Route 53
```

Comment:
Nitpicky, but maybe reword the description to better indicate that when true, this cloudformation will create the record in Route 53. The param name clearly indicates this, but the description is a little ambiguous.

Reply:
See if the latest changes sound better. If you have better text, let me know.